manifests/03-rbac-role-ns-openshift-config-managed.yaml

```diff
@@ -0,0 +1,14 @@
+kind: Role
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: console-operator
+  namespace: openshift-config-managed
+rules:
```

Comment:
Want to specify just the `console-config` specifically? I don't think we need `rbac` for all `configmaps` in the namespace.

Reply:
I tried using resourceNames: "console-config" but that was not working, then I read:
```
Notably, if resourceNames are set, then the verb must not be list, watch, create, or deletecollection. Because resource names are not present in the URL for create, list, watch, and deletecollection API requests, those verbs would not be allowed by a rule with resourceNames set, since the resourceNames portion of the rule would not match the request
```
Any other suggestions?

Reply:
@benjaminapetersen @spadgett - Any suggestions? ^^^

Reply:
ok should be fine then.